core/error.rng

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<grammar ns="urn:ietf:rfc:7807" xmlns="http://relaxng.org/ns/structure/1.0"
+    datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+
+    <start>
+
+        <element name="problem">
+
+            <zeroOrMore>
+                <ref name="foreign-attribute"/>
```

Comment:
I haven't followed the normal Libero extensibility model here, this allows any attribute or element in a different namespace to appear anywhere.